Docs for per cluster filters (#1716)
kroxylicious/kroxylicious: docs/modules/configuring/con-configuring-vc-other-settings.adoc

> [id='ref-configuring-vc-other-settings-{context}']

Reviewer: ```suggestion
[id='ref-configuring-vc-logging-{context}']
```

Reviewer: I would be tempted to keep this to logging only and change file name accordingly.

Reviewer: Assuming this should be distinct from the **Setting logs** section we have in Monitoring proxies, maybe we should rename that section?

> = Configuring other Virtual Cluster settings

Reviewer: ```suggestion
= Configuring virtual cluster logging
```

> == Per-virtual cluster logging

Reviewer: ```suggestion

```

You can enable low level logging on a per-virtual cluster basis.
The `logNetwork` property controls logging of information about requests and responses at the network level, before they've been decoded into Kafka requests and responses.
The `logFrames` property controls logging of the decoded requests and responses.

[id='con-configuring-vc-logging-{context}']
.Configuration fragment showing logging properties
[source,yaml]
----
# ... (filterDefinitions etc)
virtualClusters:
  my-cluster-proxy:
    # ... (targetCluster, clusterNetworkAddressConfigProvider, etc)
    logNetwork: true <1>
    logFrames: true <2>
    # ...
----

> <1> enable low-level network logging for this virtual cluster.

Reviewer: ```suggestion
<1> Enables low-level network logging for the virtual cluster.
```

> <2> enable low-level protocol frame logging for this virtual cluster.

Reviewer: ```suggestion
<2> Enables low-level protocol frame logging for the virtual cluster.
```
